src/components/SectionNav.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { Flex } from "@chakra-ui/react"
+import Emoji from "@/components/Emoji"
+import Translation from "@/components/Translation"
+import OldHeading from "@/components/OldHeading"
+
+export interface IProps {
```

Comment:
```suggestion
export interface SectionNavProps {
```

Reply:
```suggestion
export type SectionNavProps = {
```

Per latest TS guideline discussion, no?

Reply:
Good call, correct.